src/saltext/github/modules/github.py

```diff
@@ -1855,22 +1861,27 @@ def _query(
             headers=True,
             status=True,
             text=True,
-            hide_fields=["access_token"],
+            hide_fields=header_dict["Authorization"],
```

Comment:
**issue (blocking):** I don't think this is doing what you think it's doing. From looking at _just_ this diff, here's what I think might be the case:

**before:** we're sending a static list of strings to the `hide_fields` parameter of `http.query`. on the other side, I'm willing to bet that the `http.query` function will prevent any value with the string `"access_token"` as its field key from being shown in output.

**after:** we're sending _the value of the `Authorization` header_ as a field key. that string, which probably looks something like "`token <some encoded value>`", gets interpreted as a list by the `http.query` function on the other side. as a result, _each character of that string_ is used as a field to hide. since fields are unlikely to only be named "t", "o", "k", "e", "n", etc., this process is unlikely to hide anything at all.